support-frontend/stories/images/GridImage.stories.tsx

```diff
@@ -24,16 +25,16 @@ export default {
 	],
```

Comment:
We could probably get rid of these explainer paragraphs and use the docs parameter instead- see here for an example https://github.com/guardian/support-frontend/blob/main/support-frontend/stories/checkouts/PaymentFrequencyTabs.stories.tsx#L34 This means the info will show up on the docs page.

Reply:
Ah yes, updated, looks much neater in DocsPage template. Great suggestion.